### DISCUSSION TITLE
Add comment on behalf of another user

### DISCUSSION THREAD

**Discussion by kosist** [pnp/pnpcore#856] [General] [answered]
Is there a way to post comment on behalf of another user?
Let's say, that I'd like to migrate site from one location to another. Microsoft native migration API does not support comments migration, so I'd like to do it programmatically. But when I post comments, they are posted from authenticated user's name. Is there a way to post it on behalf of the original user name, original author of that comment?
Thanks a lot in advance.

**Comment by jansenbe** [comment, marked as answer]
@kosist : this is not possible, something I investigated in 2020 as well (as part of blog page modernization)...the API's don't have the option to set user/date. Maybe you officially ask this as design change request via official Microsoft support channels?

**Reply by kosist** [reply]
Thank you, @jansenbe!
